mvp-gate-party/src/main/kotlin/cz/tyckouni/mvpgate/party/grpc/converter/sp/SpGrpcConverterImpl.kt

```diff
@@ -15,4 +17,17 @@ class SpGrpcConverterImpl : SpGrpcConverter {
             .setName(dbEntity.name)
             .build()
     }
+
+    override fun toListGrpc(dbEntities: List<Sp>): SpsGrpc {
+        var builder = SpsGrpc.newBuilder()
+
+        builder.addAllSps(
+            dbEntities.stream().
```

Comment:
Not sure but I think that in Kotlin you can use `.map` directly without calling `stream`.

Reply:
You are right, map is enough here.